htdocs/core/modules/facture/doc/pdf_crabe.modules.php

```diff
@@ -593,7 +593,7 @@ public function write_file($object, $outputlangs, $srctemplatepath = '', $hidede
 					{
 					    $progress = pdf_getlineprogress($object, $i, $outputlangs, $hidedetails);
 					    $pdf->SetXY($this->posxprogress, $curY);
-				        $pdf->MultiCell($this->postotalht-$this->posxprogress-1, 3, $progress, 0, 'R');
+				        $pdf->MultiCell($this->postotalht-$this->posxprogress+1, 3, $progress, 0, 'R');
```

Comment:
Dont( you have the % char tha overwrite the vertical bar with this change ?

Reply:
Hi @eldy , no it's right next to the vertical bar but doesn't overwrite it